packages/MarkdownEditor-Core.package/MarkdownAttribute.class/properties.json

```diff
@@ -4,10 +4,11 @@
 		 ],
 	"classvars" : [
 		 ],
-	"commentStamp" : "fgo 5/21/2019 15:40",
+	"commentStamp" : "fgo 7/10/2019 15:23",
 	"instvars" : [
 		"startPosition",
-		"endPosition" ],
+		"endPosition",
+		"textAttribute" ],
 	"name" : "MarkdownAttribute",
```

Comment:
I am sorry for the late suggestion, but actually, the `MarkdownAttribute` would be better described as `MarkdownEmphasis` which provides `normal` (old `italic`) and `strong` (old `bold`). That way, the mapping between the Markdown domain and the Squeak implementation would be more obvious and there would be no `attribute textAttribute` naming duplication.

But if you say that renaming this class isn't worth it, I'm fine with it

Reply:
But I think `normal` would conflict with `TextEmphasis normal` which is - well - normal text...
But we'll rename `MarkdownAttribute` to `MarkdownEmphasis`.

Reply:
Maybe we could call them `weak` and `strong`. I would really like to create the mapping of spec to Smalltalk like Jakob suggested.

Reply:
If you rename it, could we also change the local variables in the test cases that are now called `attribute` to `emphasis`? If you don't want to do it, I can also do it, but then also the access is nicer with `emphasis textAttribute` instead of `attribute textAttribute`

Reply:
What about `MarkdownEmphasis from:to:` being italic and `MarkdownEmphasis strongFrom: to:` being bold?

Reply:
> If you rename it, could we also change the local variables in the test cases that are now called `attribute` to `emphasis`? If you don't want to do it, I can also do it, but then also the access is nicer with `emphasis textAttribute` instead of `attribute textAttribute`

Yeah, we could do `emphasis textAttribute`. No problem.

Reply:
> What about `MarkdownEmphasis from:to:` being italic and `MarkdownEmphasis strongFrom: to:` being bold?

That sounds good.

Reply:
Done